src/Playwright.Tests/BaseTests/HttpService.cs

```diff
@@ -16,8 +16,8 @@ public static async Task<HttpService> Register(WorkerAwareTest test)
             return await test.RegisterService("Http", async () =>
             {
                 var http = new HttpService();
-                http.Server = SimpleServer.Create(8081 + workerIndex * 2, TestUtils.FindParentDirectory("Playwright.Tests.TestServer"));
-                http.HttpsServer = SimpleServer.CreateHttps(8081 + workerIndex * 2 + 1, TestUtils.FindParentDirectory("Playwright.Tests.TestServer"));
+                http.Server = SimpleServer.Create(8907 + workerIndex * 2, TestUtils.FindParentDirectory("Playwright.Tests.TestServer"));
```

Comment:
No need for this - see #1469, it won't make a difference :-) We can rebase this PR when that one merges and it'll be fine.